configs/mainnet-asset-white-list.json

```diff
@@ -5,414 +5,382 @@
     "symbol": "ETH",
     "logoURI": "https://cryptologos.cc/logos/ethereum-eth-logo.svg?v=002",
     "decimal": 18,
-    "minimalBridgeAmount": "30000000000000000",
+    "minimalBridgeAmount": "45000000000000000",
     "bridgeFee": {
-      "in": "1590000000000000",
-      "out": "15000000000000000"
-    },
-    "sudtArgs": "0x9657b32fcdc463e13ec9205914fd91c443822a949937ae94add9869e7f2e1de8"
```

Comment:
remove `sudtArgs` field?

Reply:
`sudtArgs` is not used in the config, just listed here to show.
The show function of this file have been moved to `all-bridged-tokens.json`.